plugin/action/mask/mask.go

```diff
@@ -71,6 +71,16 @@ type Mask struct {
 	//>
 	//> Numbers of masking groups in expression, zero for mask all expression
 	Groups []int `json:"groups" required:"true"` //*
+
+	//> @3@4@5@6
+	//>
+	//> MaxCount limits the number of masked symbols in the masked output, if zero, no limit is set
+	MaxCount int `json:"max_count"` //*
+
+	//> @3@4@5@6
+	//>
+	//> ReplaceWord, if set, is used instead of asterisks for masking patterns that are of the same length or longer.
+	ReplaceWord string `json:"replace_word"` //*
```

Comment:
Let's check in `Start` that `MaxCount` and `ReplaceWord` can't both appear.
Otherwise it's controversial config and we should throw fatal.

Reply:
Done